pages/index.vue

```diff
@@ -163,7 +163,7 @@ export default {
     },
     fetchOnServer: false,
     computed: {
-        ...mapState(['sponsorsData']),
+        // ...mapState(['sponsorsData']),
```

Comment:
Hi @mattwang44, I've comment out this line and  L121, whiche can fix the error.

That error you'd mentioned is based on the setup for archive, 
I think it will works the same when we use `asyncData` in the file